ListUtil.xs

```diff
@@ -1204,6 +1204,52 @@ CODE:
     XSRETURN(items);
 }
 
+void
+sample(...)
+PROTOTYPE: @
```

Comment:
I'd say this should have a prototype of `$@` for consistency with head and tail.

Reply:
Yeah; probably best. Annoyingly that means I can't use the `ALIAS` mechanism to share the implementation with `shuffle` any more, but fair enough.

Done 56d89f2